include/igl/boundary_loop.cpp

```diff
@@ -130,7 +130,7 @@ IGL_INLINE void igl::boundary_loop(
 
 template <typename DerivedF, typename DerivedL>
 IGL_INLINE void igl::boundary_loop(
-  const Eigen::MatrixBase<DerivedF>& F,
+  const Eigen::PlainObjectBase<DerivedF>& F,
```

Comment:
This should be of type `MatrixBase` (see FAQ).

Reply:
Thanks